autoload/leaderf/Filer.vim

```diff
@@ -20,12 +20,8 @@ function! leaderf#Filer#NormalMap() abort
     let l:default_map = {}
     if get(g:, 'Lf_FilerUseDefaultNormalMap', v:true)
         let l:default_map = {
-        \   'h':             'open_parent',
```

Comment:
The mapping can be easily customized by using `g:Lf_FilerNormalMap`, so we don't need to change the default mapping.
https://github.com/tamago324/LeaderF-filer/blob/07d56ff814b602da5041525d913744eda8f3ec9e/doc/leaderf-filer.txt#L209